Dockerfile

```diff
@@ -34,6 +34,17 @@ RUN \
 
 RUN docker-php-ext-install sockets
 
+# install C extension for .mmdb reader
+# https://github.com/maxmind/libmaxminddb
+RUN wget https://github.com/maxmind/libmaxminddb/releases/download/1.4.3/libmaxminddb-1.4.3.tar.gz
```

Comment:
We should combine all this `RUN` statements to better manage the image caching. Take a look at the `RUN` statement where we install `phpredis` and `swoole` extensions.